Remove documentation of dashboard durable object bugs (#2747)
cloudflare/cloudflare-docs: products/workers/src/content/learning/using-durable-objects.md

[Wrangler tail](/cli-wrangler/commands#tail) logs from requests that are upgraded to WebSockets are delayed until the WebSocket is closed.  Wrangler tail should not be connected to a script that you expect will receive heavy volumes of traffic.

> [Wrangler dev](/cli-wrangler/commands#dev) opens up a tunnel from your local development environment to Cloudflare's edge, letting you test your Durable Objects code in the Workers environment as you write it.

Reviewer: ```suggestion
[Wrangler dev](/cli-wrangler/commands#dev) establishes a tunnel from your local development environment to Cloudflare's edge, allowing you to test your Worker and Durable Objects as they are developed.
```

> The Workers editor in [the Cloudflare dashboard](https://dash.cloudflare.com/) allows you to interactively edit and preview your script and any Durable Objects that it both implements and talks to.

Reviewer: ```suggestion
The Workers editor in [the Cloudflare dashboard](https://dash.cloudflare.com/) allows you to interactively edit and preview your Worker and any Durable Objects that it may incorporate.
```

Author: "may incorporate" is perhaps a bit overly broad -- it really only works for DOs that are both implemented by and bound to the script being previewed. You cannot talk to DOs implemented by other scripts in preview. Do you think this updated version still conveys that? Or is there a better way to word that? Or do you just not think that's important to convey?

Reviewer: Ha, fair. I was trying to dedupe the wording because defining, exporting, and referencing the DOs are just necessary usage requirements explained elsewhere. Can't think of a better wording... feel free to disregard!

Author: Yeah the wording is a bit awkward, but I do think that it's worth conveying the requirement. I'll pull in the s/script/Worker change though.

Author: And maybe saying "both exports and binds to" is a little more clear because at least it uses the more official terminology. I'll switch to that.

Author: I tried elaborating a bit on that point, let me know if it doesn't make sense.

Reviewer: Looks good!
